detekt.yml

```diff
@@ -1,6 +1,3 @@
-# turn on ALL checks
-failFast: true
```

Comment:
did detekt change the default? we had to explicitly set it to true in the past

Reply:
Yes, it was giving me and warning on builds saying to use the CLI flag instead

Reply:
https://github.com/arturbosch/detekt/pull/1479